app/jobs/invoices/generate_files_and_notify_job.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Invoices
-  class GeneratePdfAndNotifyJob < ApplicationJob
+  class GenerateFilesAndNotifyJob < ApplicationJob
```

Comment:
I think you can't rename jobs because there could be `GeneratePdfAndNotifyJob` jobs enqueued and after the deployed they'll all fail because the job class is not found.

Maybe we can simply create an alias with the old name but usually, we just keep the old class and make it call the new class... _forever_.